src/libraries/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs

```diff
@@ -412,6 +417,11 @@ public partial interface ICustomQueryInterface
     {
         System.Runtime.InteropServices.CustomQueryInterfaceResult GetInterface(ref System.Guid iid, out System.IntPtr ppv);
     }
+    public partial interface IDynamicInterfaceCastable
+    {
+        bool IsInterfaceImplemented(System.RuntimeTypeHandle interfaceType, bool isDirectCast);
```

Comment:
```suggestion
        bool IsInterfaceImplemented(System.RuntimeTypeHandle interfaceType, bool throwIfNotImplemented);
```
This is the argument name from the API review discussion. I think it better explains what the argument does.

Reply:
Ah, yeah, agreed. I had renamed before we did the re-review and forgot to go back to change it.